ldap/servers/slapd/slap.h

```diff
@@ -40,12 +40,11 @@ static char ptokPBE[34] = "Internal (Software) Token        ";
 #define SLAPD_EXEMODE_ARCHIVE2DB       5
 #define SLAPD_EXEMODE_DBTEST           6
 #define SLAPD_EXEMODE_DB2INDEX         7
-#define SLAPD_EXEMODE_REFERRAL         8
-#define SLAPD_EXEMODE_SUFFIX2INSTANCE  9
-#define SLAPD_EXEMODE_PRINTVERSION    10
-#define SLAPD_EXEMODE_UPGRADEDB       11
-#define SLAPD_EXEMODE_DBVERIFY        12
-#define SLAPD_EXEMODE_UPGRADEDNFORMAT 13
+#define SLAPD_EXEMODE_SUFFIX2INSTANCE  8
+#define SLAPD_EXEMODE_PRINTVERSION     9
+#define SLAPD_EXEMODE_UPGRADEDB       10
+#define SLAPD_EXEMODE_DBVERIFY        11
+#define SLAPD_EXEMODE_UPGRADEDNFORMAT 12
```

Comment:
I worry a bit here that we may break binary compatibility because of the reordering

Reply:
Yes I understand your concern, let me revert the reordering and mark it as deprecated.

Reply:
I had similar concern but IIUC it is only used internally in the server (no CLI). I would also vote to keep the current ordering (deprecating the flag with a comment for example).